packages/ui/tsconfig.native.json

```diff
@@ -15,6 +15,7 @@
     "**/*.native.ts",
     "**/*.native.tsx",
     "native.ts",
+    "*.native.tsx",
```

Comment:
I added this as I added a `components` folder to `packages/ui/sheet` to help organise the code. 

I'm not sure if this is the best way of doing it. We could perhaps remove this code from the includes:
```
"**/*.native.ts",
 "**/*.native.tsx",
```
Replacing it with 
```
 "*.native.ts",
  "*.native.tsx",
```

Reply:
Until I changed it, it was complaining about the `components` not being included. I will do some tests with it and change if necessary. Thanks 👍

Reply:
I tested this and the original pattern did catch it, it must have been the IDE TS server fooling me